leafsmart-front-end/src/components/Favourites.js

```diff
@@ -1,39 +1,30 @@
 import React, { useEffect, useState } from "react";
-import axios from "axios";
+import FavouritesCard from "./FavouritesCard";
```

Comment:
This appearing twice caused an error! I needed to delete it to get the app to fire. Can one of the duplicate statements be removed?

Reply:
Sorry which states are you looking at?